src/rosecommon/include/enums.h

```diff
@@ -33,10 +33,10 @@ enum EquippedPosition : uint8_t {
 };
 
 enum MoveMode : uint8_t {
-    WALK = 0,
-    RUN = 1,
-    DRIVE = 2,
-    RIDEON = 4
+    WALK = 2,
+    RUN = 3,
+    DRIVE = 4,
+    // RIDEON = 4
```

Comment:
I'm not sure if this is correct since I believe the old values were from the client. Need to check.

Reply:
Based on tests and the client binary I'm using, those were the actual values.
But you're right if we're talking about the client source we have under dev-osrose.

Reply:
Have we checked that?

Reply:
Well it is working(checked that in-game ofc). But it against the ENUM inside the client :D